package ru.sylas.exceptions

class UserAlreadyCreatedException():Exception("Имя пользователя занято")